examples/pytorch/image_recognition/segment_anything/main.py

```diff
@@ -0,0 +1,349 @@
+            (self.image_encoder.img_size, self.image_encoder.img_size),
+            mode="bilinear",
+            align_corners=False,
+        )
+        masks = masks[..., : input_size[0], : input_size[1]]
+        masks = F.interpolate(masks, original_size, mode="bilinear", align_corners=False)
+        return masks
+
+    def preprocess(self, x: torch.Tensor) -> torch.Tensor:
+        # Normalize colors
+        x = (x - self.pixel_mean) / self.pixel_std
+
+        # Pad
+        h, w = x.shape[-2:]
+        padh = self.image_encoder.img_size - h
+        padw = self.image_encoder.img_size - w
+        x = F.pad(x, (0, padw, 0, padh))
+        return x
+
+
+def eval_func(model):
+    device = 'cpu'
+    metric = torchmetrics.Dice(ignore_index=0).to(device) #Ignore background
+    list_of_metrics = []
+    
+    for i, (input, gt) in enumerate(eval_dataloader):
+        preds = model(input['image'],
+                      input['prompt'],
+                      input["original_size"],
+                      input["input_size"],
+                      input["ground_truth_mask"])
+        labels = gt
+        result = metric(preds.reshape(-1), labels.reshape(-1).int())
+        list_of_metrics.append(result)
+
+    return np.array(list_of_metrics).mean()
+
+
+def validate(eval_dataloader, model, args):
+    model.eval()
+    device = 'cpu'
+    metric = torchmetrics.Dice(ignore_index=0).to(device)
+    list_of_metrics = []
+
+    with torch.no_grad():
+        for i, (input, gt) in enumerate(eval_dataloader):
+            preds = model(input['image'],
+                        input['prompt'],
+                        input["original_size"],
+                        input["input_size"],
+                        input["ground_truth_mask"])
+            labels = gt
+            result = metric(preds.reshape(-1), labels.reshape(-1).int())
+            list_of_metrics.append(result)
+
+    print("Average Dice Score: " + str(np.array(list_of_metrics).mean()) )
+    return
+    
+    
+# Start PTQ
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='PyTorch VOC Training')
+    parser.add_argument("--pretrained_weight_location", default='./sam_vit_b_01ec64.pth', type=str,
+                        help='Location of the image encoder pretrained weights')
+    parser.add_argument('--voc_dataset_location', default='./voc_dataset/VOCdevkit/VOC2012/', type=str,
+                        help='Path of the VOC Dataset')
+    parser.add_argument("--tuned_checkpoint", default='./saved_results', type=str, metavar='PATH',
+                        help='path to checkpoint tuned by Neural Compressor (default: ./)')
+    parser.add_argument("--tune", action='store_true',
+                        help='Apply INT8 quantization or not')   
+    parser.add_argument('--int8', action='store_true',
+                        help='for benchmarking/validation using quantized model')
+    parser.add_argument('--dice', action='store_true',
```

Comment:
It would be better to use `--accuracy` for accuracy measurement argument, for consistency.